src/main/features/claimant-response/views/confirmation.njk

```diff
@@ -17,7 +17,7 @@
           {% elseif repaymentPlanOrigin === 'COURT' %}
             {{ t('You’ve signed a settlement agreement') }}
           {% elseif claim.countyCourtJudgmentIssuedAt %}
```

Comment:
Are you not meant to be looking at countyCourtJudgmentRequestedAt instead of issuedAt?